pkgs/development/python-modules/pyautogui/fixes.patch

```diff
@@ -0,0 +1,30 @@
+diff --git a/pyautogui/__init__.py b/pyautogui/__init__.py
+index ec7d097..443b146 100644
+--- a/pyautogui/__init__.py
++++ b/pyautogui/__init__.py
+@@ -216,9 +216,9 @@ try:
+ 
+     @raisePyAutoGUIImageNotFoundException
+     def locateOnWindow(*args, **kwargs):
+-        return pyscreeze.locateOnWindow(*args, **kwargs)
++        return pyscreeze.locateOnScreen(*args, **kwargs)
+ 
+-    locateOnWindow.__doc__ = pyscreeze.locateOnWindow.__doc__
++    locateOnWindow.__doc__ = pyscreeze.locateOnScreen.__doc__
```

Comment:
I think this patch can be upstreamed :thinking: .

Reply:
Someone already did this but got no response

https://github.com/asweigart/mouseinfo/pull/7

https://github.com/asweigart/pyautogui/pull/503

Reply:
We generally prefer to use the patches from PRs, even if the PR itself is not merged yet.

Also, I am not seeing this specific change on the PRs, so I still think this `pyscreeze` change is worth it upsteraming.